Program.cs

```diff
@@ -0,0 +1,238 @@
+﻿using RefrigeratorExercise.Etities;
+using System;
+using System.Globalization;
+using System.Runtime.CompilerServices;
+using System.Security.Cryptography.X509Certificates;
+
+public class Program
+{
+    private static Refrigerator refrigerator;
+
+    public IEnumerable<Shelf> Shelves { get; private set; }
```

Comment:
This is really weird, we should only interact through the refrigerator interface.